Explictly set height instead of using aspectRatio [#167489026]

dougmartin opened the pull request:
This depends on the following Lara PR to be merged and released:

https://github.com/concord-consortium/lara/pull/462

dougmartin commented:
@knowuh This removes code so I thought it would be right up your alley.

dougmartin commented:
@knowuh Could you re-review?  We created a new explicit height message in the lara iframe api.

knowuh commented:
Where is the LARA interactive API documentation being stored these days?  Is it updated?

dougmartin commented:
@knowuh I added back code that I mistakenly deleted and then I added a support library that Ethan can use the project landing page to support the dynamic sizing.  I will update the LARA API docs next.

scytacki commented:
The API is documented here: http://concord-consortium.github.io/lara-interactive-api/docs/ But it hasn't been updated with this new height message.

Also there is documentation about interactive sizing in the LARA Authoring User Manual google doc.
